### DISCUSSION TITLE
New Uptane logo

### DISCUSSION THREAD

**Discussion by hexsecs** [uptane/uptane.github.io#74] [Ideas]
Here is a proposal for a simplified modern looking Uptane logo. What do you think?
![image](https://github.com/uptane/uptane.github.io/assets/89217603/8ccfd85e-c8e8-4c2a-a777-cd55c324847e)

**Comment by jhdalek55** [comment]
I like it, Phil. My only suggestion is to reverse the order and put the car on the right. I think the project name should be the dominant item. Also, perhaps use a slightly thicker serif font just to give more weight to the name. Otherwise, I think its great.

**Reply by hexsecs** [reply]
Regarding icon before text, I was following the format here...
https://github.com/cncf/artwork/blob/master/examples/graduated.md#tuf-logos
Here is the reversed order.
![uptane-horizontal-color](https://github.com/uptane/uptane.github.io/assets/89217603/6242f083-dcbe-4238-88ef-feefdf30c7ac)



Regarding weight and font, I'm open to feedback. Honestly, I prefer an even lighter font weight. The proposal is using the 400 weight. Here it is (without the lock icon) plus/minus 100 weight from normal.
<img width="455" alt="image" src="https://github.com/uptane/uptane.github.io/assets/89217603/471c0066-4503-4966-b2c5-ed48a7122e59">
<img width="478" alt="image" src="https://github.com/uptane/uptane.github.io/assets/89217603/27014079-e855-41a5-832a-ba19914b4c5f">
<img width="483" alt="image" src="https://github.com/uptane/uptane.github.io/assets/89217603/8972f410-56bb-474c-9b17-66b64cfea70b">

**Comment by jhdalek55** [comment]
Ok. Understood about the logo. I guess it makes sense to follow the same pattern. As for the font, I'll leave it to you and Ira, but I do think a thicker font is called for. I'd be fine even with a step up to Regular 400.

Just my 2 cents. It's a good workable logo as is. 

Lois

**Comment by hexsecs** [comment]
Some slight variations with fonts...

### Montserrat - Normal
![uptane-horizontal-color-montserrat](https://github.com/uptane/uptane.github.io/assets/89217603/c5568f54-b05a-4157-94d4-34038ec5e932)

### Montserrat - Medium
![uptane-horizontal-color-montserrat-medium](https://github.com/uptane/uptane.github.io/assets/89217603/b3ae6fa7-3947-4bbb-b32d-81b3e5d1da37)

### Raleway - Regular 400
![uptane-horizontal-color](https://github.com/uptane/uptane.github.io/assets/89217603/15f25290-0c4b-44e5-82ac-8898588a1adb)

**Reply by trishankkarthik** [reply]
First of all, great work, thanks! Agree with @plapczyn: could we try a light font?

**Reply by hexsecs** [reply]
@trishankkarthik : even lighter weight than the above?

**Reply by trishankkarthik** [reply]
Yes, even lighter than normal

**Reply by hexsecs** [reply]
### Montserrat - Light
![uptane-horizontal-color-montserrat-light](https://github.com/uptane/uptane.github.io/assets/89217603/e41d3945-305b-4ccc-bbc3-d26a2f6a0cc2)

**Reply by hexsecs** [reply]
We could perhaps play around with 'ota' being inside the word "uptane". Might be getting a little too busy.
![uptane-horizontal-color-montserrat-light-ota](https://github.com/uptane/uptane.github.io/assets/89217603/fb6a79a1-22c8-4922-ac36-1729c5d85b21)
![uptane-horizontal-color-montserrat-light-ota-blk](https://github.com/uptane/uptane.github.io/assets/89217603/2ad5f2f3-f496-48f6-a239-00f4d11cb0e3)

**Comment by jhdalek55** [comment]
I like the OTA illusion, but agree that it might get a bit too busy. I have
always liked the lock in the letter A though and would like that retained.

**Reply by trishankkarthik** [reply]
I'm a **big** fan of the **Montserrat - Light** font. Maybe we could see how the logo works with a similarly-weighted line, but I can live without it.

**Comment by hexsecs** [comment]
The group voted for the Montserrat - Light version
